sources/api/migration/migrations/v0.3.2/migrate-admin-container/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "migrate-admin-container"
+version = "0.1.0"
+authors = ["Erikson Tung <etung@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2018"
+publish = false
+
+[dependencies]
+migration-helpers = { path = "../../../migration-helpers" }
+schnauzer = { path = "../../../../schnauzer" }
+serde_json = "1.0"
+snafu = "0.6"
```

Comment:
Why do we need these?  We don't seem to use them directly, and they should be brought in by the migration-helpers dependency if it needs them.

Reply:
Do we need schnauzer?  I see that's still in there.